src/network/core/packet.h

```diff
@@ -40,6 +40,8 @@ typedef uint8_t  PacketType; ///< Identifier for the packet
  *     (year % 4 == 0) and ((year % 100 != 0) or (year % 400 == 0))
  */
 struct Packet {
+	static constexpr size_t EncodedLengthOfPacketSize() { return sizeof(PacketSize); }
+	static constexpr size_t EncodedLengthOfPacketType() { return sizeof(PacketType); }
```

Comment:
Excellent :D Didn't know this many places were using `sizeof()` already. This is much more clear what is going on :)